.evergreen.yml

```diff
@@ -1,7 +1,11 @@
 stepback: false
 exec_timeout_secs: 5400
 ignore:
```

Comment:
I guess we can also just ignore all *.md files, I don't see why we'd ever want to run our whole CI when they change, no strong preference though, tell me if you have a stronger opinion on this